authors/apps/profiles/views.py

```diff
@@ -175,3 +178,20 @@ def get(self, request):
             'followers': followers,
             'status': status.HTTP_200_OK}
         return Response(data=msg, status=status.HTTP_200_OK)
+
+
+class UserProfileListAPIView(generics.ListAPIView):
+    """
+    This class return all user profiles in the available in the database
+    """
+    permission_classes = (permissions.IsAuthenticated,)
+    serializer_class = ProfileSerializer
+
+    def get(self, request):
+        profiles = Profile.objects.all()
+        serializer = self.serializer_class(profiles, many=True)
+        response_data = {
+            "profiles": serializer.data,
+            "message": "successfully returned all users profiles"
```

Comment:
can the message here be changed to start with a capital S

Reply:
Good suggestion. thanks